src/config.js

```diff
@@ -147,12 +147,12 @@ function isPortable () {
     return false
   }
 
-  let portablePathExists = false
+  let pathExists = false
   try {
-    portablePathExists = !!fs.statSync(PORTABLE_PATH)
+    pathExists = !!fs.accessSync(PORTABLE_PATH, fs.constants.R_OK | fs.constants.W_OK)
```

Comment:
I was honestly in doubt myself when reading the docs. It's so strange that the purpose of this method is to throw an error or just do nothing?? 😄